WcaOnRails/spec/rails_helper.rb

```diff
@@ -52,6 +52,9 @@
     if defined? page.driver.add_header
       page.driver.add_header("Accept-Language", "en-US", permanent: true)
     end
+    if (defined? page) && page.has_css?("button #cookies-eu-ok")
```

Comment:
With the "only render if prod" change from above, is this block still necessary?

Reply:
Absolutely not! Was going to change it yesterday but internet had issues